esa/saw.py

```diff
@@ -791,7 +791,10 @@ def get_jacobian(self, full=False):
         jidfile = tempfile.NamedTemporaryFile(mode='w', delete=False)
         jidfile_path = Path(jidfile.name).as_posix()
         jidfile.close()
-        cmd = f'SaveJacobian("{jacfile_path}","{jidfile_path}",M,R);'
+        if Polar:
```

Comment:
Then you can pass the `JacForm` directly to the aux command, which also makes it easier to test

Reply:
sounds good. have you pushed?

Reply:
Thought I did, but looks like it didn't go through. I'll post a comment again when I push it

Reply:
I updated the file again, should show now.